scripts/typescript_checks.py

```diff
@@ -189,185 +106,60 @@
-    'core/templates/pages/exploration-editor-page/editor-tab/unresolved-answers-overview/unresolved-answers-overview.component.ts',
     'core/templates/pages/exploration-editor-page/exploration-editor-page.component.spec.ts',
     'core/templates/pages/exploration-editor-page/exploration-editor-page.component.ts',
     'core/templates/pages/exploration-editor-page/exploration-editor-page.import.ts',
-    'core/templates/pages/exploration-editor-page/exploration-objective-editor/exploration-objective-editor.component.spec.ts',
-    'core/templates/pages/exploration-editor-page/exploration-objective-editor/exploration-objective-editor.component.ts',
     'core/templates/pages/exploration-editor-page/exploration-save-and-publish-buttons/exploration-save-and-publish-buttons.component.spec.ts',
     'core/templates/pages/exploration-editor-page/exploration-save-and-publish-buttons/exploration-save-and-publish-buttons.component.ts',
-    'core/templates/pages/exploration-editor-page/exploration-title-editor/exploration-title-editor.component.spec.ts',
-    'core/templates/pages/exploration-editor-page/exploration-title-editor/exploration-title-editor.component.ts',
     'core/templates/pages/exploration-editor-page/feedback-tab/feedback-tab.component.spec.ts',
     'core/templates/pages/exploration-editor-page/feedback-tab/feedback-tab.component.ts',
-    'core/templates/pages/exploration-editor-page/history-tab/history-tab.component.spec.ts',
-    'core/templates/pages/exploration-editor-page/history-tab/history-tab.component.ts',
-    'core/templates/pages/exploration-editor-page/history-tab/services/compare-versions.service.spec.ts',
-    'core/templates/pages/exploration-editor-page/history-tab/services/compare-versions.service.ts',
     'core/templates/pages/exploration-editor-page/improvements-tab/improvements-tab.component.spec.ts',
     'core/templates/pages/exploration-editor-page/improvements-tab/improvements-tab.component.ts',
-    'core/templates/pages/exploration-editor-page/improvements-tab/needs-guiding-responses-task.component.spec.ts',
-    'core/templates/pages/exploration-editor-page/improvements-tab/needs-guiding-responses-task.component.ts',
-    'core/templates/pages/exploration-editor-page/improvements-tab/services/improvement-modal.service.spec.ts',
-    'core/templates/pages/exploration-editor-page/improvements-tab/services/improvement-modal.service.ts',
-    'core/templates/pages/exploration-editor-page/modal-templates/exploration-metadata-modal.component.spec.ts',
-    'core/templates/pages/exploration-editor-page/modal-templates/exploration-metadata-modal.component.ts',
-    'core/templates/pages/exploration-editor-page/modal-templates/exploration-save-modal.component.spec.ts',
-    'core/templates/pages/exploration-editor-page/modal-templates/exploration-save-modal.component.ts',
+    'core/templates/pages/exploration-editor-page/exploration-editor-page.module.ts',
```

Comment:
Why are we adding this file to the exclusion list since this PR is supposed to be making typescript checks stricter?

Reply:
Done